ROADMAP.md

```diff
@@ -298,14 +300,14 @@ Status | Command | Description
 
 Status | Command | Description
 ---|--------|------------------------------
-:warning:   | "{char}	        | use register {char} for the next delete, yank, or put
+:warning: :red_circle:  | "{char}	        | use register {char} for the next delete, yank, or put
```

Comment:
@aminroosta you implement these features and I'm seeing them working well. Is there any hidden issue with it which makes you mark it as ⚠️ ?